src/pages/Home.tsx

```diff
@@ -1,14 +1,51 @@
 import { MainLayout } from '../layouts/MainLayout';
 import { Search } from '../components/Search';
-import { Box } from 'grommet';
+import { Grid, Container } from '@mui/material';
+import CityCarousel from '../container/home/CityCarousel';
+import berlin from '../images/berlin.jpeg';
+import bogota from '../images/bogota.jpeg';
+import sydney from '../images/sydney.jpeg';
+import taipei from '../images/taipei.jpeg';
+import ConferenceCarousel from '../container/home/ConferenceCarousel';
 
+const cities = [
```

Comment:
I think you re not using this data in here, but in the conatiner components, right?